### DISCUSSION TITLE
Add more options for Marker Node appearance

### DISCUSSION THREAD

**Discussion by LordMcMutton** [godotengine/godot-proposals#9335] [Editor]
A pretty basic idea:

Marker2D and Marker3D Nodes are very useful for visualizing where a Node is in the editor, but the axes are rather small and can be hard to see

I'd like to see an addition to them that allows for various shape options, like circles or diamonds, in addition to colors. 

Aside from making the Marker Nodes more visible, they would allow for a bit of visual categorization

That said, if the methods used to display the axes can't create filled-in shapes, the visibility issue may still remain, in which case, perhaps just being able to make the axes thicker would achieve the same affect

**Comment by Calinou** [comment]
Making the axes thicker means that they'd be drawn using polylines, which scale with viewport zoom and isn't always desired. They could become huge in pixel art projects at high zoom levels (working at 800% or 1600% is common in these projects).

We can work around this by drawing two thin lines at slightly different positions to simulate a thick line, but that doesn't scale with viewport zoom. It may not look great at higher zoom levels though.

**Reply by LordMcMutton** [reply]
That would be a less desirable version, then?

Something like Unity does, with little sprites would work as well

examples:
![image](https://github.com/godotengine/godot-proposals/assets/93609836/706f34a1-ed36-4be9-aff4-0c133087bcb8)
![image](https://github.com/godotengine/godot-proposals/assets/93609836/f91feb5e-517c-40f3-979c-be984ed16ab1)
